distribution/bin/check-licenses.py

```diff
@@ -20,6 +20,7 @@
 import sys
 from html.parser import HTMLParser
 import argparse
+#import json
```

Comment:
let's remove this line if it is not necessary

Reply:
done!